### PR TITLE
Crawler result (An End-to-end System for Automatic Characterization of Iba1 Immunopositive Microglia in Whole Slide Imaging: Data)

### PR DESCRIPTION
**Pull request by gi114**
## Description
Created An End-to-end System for Automatic Characterization of Iba1 Immunopositive Microglia in Whole Slide Imaging: Data

## Checklist

Mandatory files and elements:
- [x] A `README.md` file, at the root of the dataset
- [x] A `DATS.json` file, at the root of the dataset
- [x] If configuration is required (for instance to enable a special remote), a `config.sh` script at the root of the dataset
- [x] A DOI (see instructions in [contribution guide](https://github.com/CONP-PCNO/conp-dataset/blob/master/.github/CONTRIBUTING.md), and corresponding badge in `README.md`

Functional checks:
- [x] Dataset can be installed using DataLad, recursively if it has sub-datasets
- [x] Every data file has a URL
- [x] Every data file can be retrieved or requires authentication
- [x] `DATS.json` is a valid DATs model
- [ ] If dataset is derived data, raw data is a sub-dataset

### CODE REVIEW

File: scripts/Crawlers/FrdrCrawler.py [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
+                    "keywords": list(map(lambda x: {"value": x}, metadata["dc:subject"])),
+                    "distributions": [
+                        {
+                            "formats": [
+                                file_format.upper()
+                                for file_format in files_types
+                                # Do not modify specific file formats.
+                                if files_types not in ["NIfTI", "BigWig"]
+                            ],
+                            "size": str(ds_size),
```
Comment (joeyzhou98) [inline scripts/Crawlers/FrdrCrawler.py:311]: `ERROR    scripts.dats_validator.validator:validator.py:57 2 Validation error in distributions.0.size: '14892864859' is not of type 'number'
`
Change to `"size": int(ds_size),`

File: scripts/Crawlers/FrdrCrawler.py [hunk 2]
```diff
@@ -0,0 +1,532 @@
...
+            metadata = dataset["metadata"]["frdr:frdr"]
+
+            # TODO: to be updated: we are now testing 1 dataset
+            if "An End-to-end System for Automatic Characterization of Iba1 Immunopositive Microglia in Whole Slide Imaging: Data" in metadata["dc:title"]:
+
+                files_types, ds_size, globus_path = \
+                    self.get_all_files_description(metadata["frdr:globusEndpointName"],
+                                                   metadata["frdr:globusEndpointPath"])
+
+                frdr_dois.append(
```
Comment (joeyzhou98) [inline scripts/Crawlers/FrdrCrawler.py:281]: `ERROR    scripts.dats_validator.validator:validator.py:57 1 Validation error in distributions.0.access: 'landingPage' is a required property
`
Add a `landingPage` property.
See https://github.com/CONP-PCNO/conp-dataset/blob/master/scripts/Crawlers/OSFCrawler.py#L109

File: scripts/crawl.py [hunk 3]
```diff
@@ -4,10 +4,10 @@
 import os
 import sys
 import traceback
-sys.path.append(os.path.abspath(os.path.join(os.path.expanduser("~"), "conp-dataset")))
-sys.path.append(os.path.join("/tmp", "conp-dataset"))
+sys.path.append(os.path.abspath("/data/giulia_crawler/conp-dataset"))
```
Comment (joeyzhou98) [inline scripts/crawl.py:7]: Revert this change to 
`sys.path.append(os.path.abspath(os.path.join(os.path.expanduser("~"), "conp-dataset")))	`

File: scripts/crawl.py [hunk 4]
```diff
@@ -34,7 +34,7 @@ def parse_args():
     config_path = args.config_path
     if not config_path:
         config_path = os.path.join(
-            os.path.expanduser("~"), ".conp_crawler_config.json")
+            os.path.expanduser("~"), ".conp_crawler_config_gi114.json")
```
Comment (joeyzhou98) [inline scripts/crawl.py:37]: Revert change to `            os.path.expanduser("~"), ".conp_crawler_config.json")	`

File: scripts/crawl.py [hunk 5]
```diff
@@ -64,15 +64,20 @@ def parse_args():
...
-        ZenodoCrawler.run()
+        # if verbose:
+        #     print("==================== Zenodo Crawler Running ====================" + os.linesep)
+        # ZenodoCrawler = ZenodoCrawler(github_token, config_path, verbose, force)
+        # ZenodoCrawler.run()
+        #
+        # if verbose:
+        #     print(os.linesep + "==================== OSF Crawler Running ====================" + os.linesep)
+        # OSFCrawler = OSFCrawler(github_token, config_path, verbose, force)
+        # OSFCrawler.run()
```
Comment (joeyzhou98) [inline scripts/crawl.py:75]: Uncomment these

**Comment by gi114** [comment]
@joeyzhou98 
I made the changes requested, let me know if that works. The landingPage should be the https page of the dataset, am I right?

**Comment by joeyzhou98** [comment]
> @joeyzhou98
> I made the changes requested, let me know if that works. The landingPage should be the https page of the dataset, am I right?

Yes

**Comment by cmadjar** [comment]
@gi114 this is failing circle CI and has conflicts with the master branch
